packages/embark-utils/src/index.js

```diff
@@ -7,6 +7,26 @@ const {canonicalHost, defaultCorsHost, defaultHost, dockerHostSwap, isDocker} =
 const {findNextPort} = require('./network');
 const logUtils = require('./log-utils');
 const toposortGraph = require('./toposort');
+import { unitRegex } from './constants';
+import * as AddressUtils from './addressUtils';
+const web3 = require("web3");
```

Comment:
I wouldn't require web3 at the top of the file. `web3` is the reason we started putting requires inside the function, because some processes needed to use an util function like `checkIsAvailable`, but it ended up loading `web3` (it's not cached in that case because it's another process), so it slowed down the whole process.